glossary.yml

```diff
@@ -6078,6 +6086,11 @@
     term: የተተረጎመ ቋንቋ
     def: >
       በቀጥታ በኮምፒተር የማይተገበር የከፍተኛ ደረጃ ቋንቋ ፣ ግን ይልቁንስ የሚከናወነው የፕሮግራም መመሪያዎችን በሚተረጎም በ [አስተርጓሚ](#interpreter) ነው በመብረር ላይ ወደ ማሽን ትዕዛዞች ፡፡
+  uk:
+    term: "інтерпретована мова"
+    def: >
+      Мова високого рівня, яка не виконується напряму комп'ютером. Натомість її інструкції 
+      перекладаються у машинний код безпосередньо під час виконання за допомогою [інтерпретатора](#interpreter).
```

Comment:
безпосередньо - seems unnecessary

Reply:
But it used in "компільована мова", and I think it's important to translate "on the fly". Otherwise, a beginner may misinterpret that with "compile and run" when compile is done just before run.

Reply:
I will leave this as it was